### DISCUSSION TITLE
How can I find out the Google Play Billing Library version?

### DISCUSSION THREAD

**Discussion by bloggerklik** [hyochan/react-native-iap#2049] [Q&A] [answered]
I am using version 8.0.10 of react-native-iap package. What is the Google Play Billing Library version in this version? How can I find out the Google Play Billing library version? Thanks.

**Comment by andresesfm** [comment, marked as answer]
open `android/build.gradle` inside your `node_modules/react-native-iap`

**Reply by bloggerklik** [reply]
Which of these?

`dependencies {
  playImplementation 'com.android.billingclient:billing:4.0.0'`

or

`android {
  defaultConfig {
    versionCode 3
    versionName "3.0.0"
  }`

**Reply by okcompewter** [reply]
The deadline to move to Google Play Billing Library version 6 is coming up in a few months. How can I tell which library version is supported by the react-native-iap (I'm on 12.10.7)?

Looking at Google's migration guide here:
https://developer.android.com/google/play/billing/migrate-gpblv7#update-google

Do I just have to add this line to my app/build.gradle file (and I could I jump right to v7)?

```
dependencies {
    def billingVersion = 7.0.0

    implementation "com.android.billingclient:billing:$billingVersion"
}
```

**Reply by okcompewter** [reply]
Found the version is listed in a gradle property (`/android/gradle.properties`):
https://github.com/dooboolab-community/react-native-iap/blob/9377af95b47443ceff8fba866abdbb61d62cebd1/android/gradle.properties#L9C1-L9C34

**Comment by andresesfm** [comment]
`com.android.billingclient:billing`

**Reply by bloggerklik** [reply]
Thank you :)
